packages/editor/src/components/post-taxonomies/hierarchical-term-selector.js

```diff
@@ -319,15 +318,16 @@ class HierarchicalTermSelector extends Component {
 			const id = `editor-post-taxonomies-hierarchical-term-${ term.id }`;
 			return (
 				<div key={ term.id } className="editor-post-taxonomies__hierarchical-terms-choice">
-					<input
+					<CheckboxControl
 						id={ id }
-						className="editor-post-taxonomies__hierarchical-terms-input"
```

Comment:
We need to remove all the CSS and change all the e2e tests that used this `className` as a selector.

Reply:
🤪

Reply:
attempt one